styles/style-main.css

```diff
@@ -0,0 +1,95 @@
+.navbar {
+  display: flex;
+  justify-content: center;
+  background-color: beige;
+}
+
+.navbar ul {
+  display: flex;
+  align-self: center;
+  width: 40vw;
+  padding: 10px;
+  justify-content: center;
+}
+
+.navbar ul > * {
+  flex: 1 1 25%;
+  list-style: none;
+  text-align: center;
+  font-size: 30px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+a {
+  text-decoration: none;
+}
+
+main {
+  display: block;
+}
+
+section {
+  display: flex;
+}
+
+.headline-left {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 60%;
+}
+
+.headline-name {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
- [ ] Kindly use the correct font family, the one for this would be the [Playfair Display, which you can find on google fonts](https://fonts.google.com/specimen/Playfair+Display)